insights/parsers/journal_all.py

```diff
@@ -5,11 +5,19 @@
 
 from .. import Syslog, parser
 from insights.specs import Specs
+from insights.util import deprecated
 
 
 @parser(Specs.journal_all)
 class JournalAll(Syslog):
     """
+
+    .. warning::
+
+        This parser is deprecated, please use
```

Comment:
There is a mismatch between this warning msg and the actual deprecation msg.

The warning msg suggest to use `insights.parsers.journalctl.JournalSinceBoot` where as the deprecation msg suggest's `insights.parsers.auditctl.AuditdStatus`

Reply:
Sorry for that and updated